specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -524,6 +629,86 @@
           "minimum": 1,
           "maximum": 102400,
           "description": "The maximum size of the share, in gigabytes. Must be greater than 0, and less than or equal to 5TB (5120). For Large File Shares, the maximum size is 102400."
+        },
+        "enabledProtocols": {
+          "type": "string",
+          "enum": [
+            "SMB",
+            "NFS"
+          ],
+          "x-ms-enum": {
+            "name": "EnabledProtocols",
+            "modelAsString": true
+          },
+          "description": "The authentication protocol that is used for the file share. Can only be specified when creating a share.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "rootSquash": {
+          "type": "string",
+          "enum": [
+            "NoRootSquash",
+            "RootSquash",
+            "AllSquash"
+          ],
+          "x-ms-enum": {
+            "name": "RootSquashType",
+            "modelAsString": true
+          },
+          "description": "The property is for NFS share only. The default is NoRootSquash."
+        },
+        "version": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The version of the share."
+        },
+        "deleted": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Indicates whether the share was deleted."
+        },
+        "deletedTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The deleted time if the share was deleted."
+        },
+        "remainingRetentionDays": {
+          "type": "integer",
+          "readOnly": true,
+          "description": "Remaining retention days for share that was soft deleted."
+        },
+        "accessTier": {
+          "type": "string",
+          "enum": [
+            "TransactionOptimized",
+            "Hot",
+            "Cool",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "ShareAccessTier",
+            "modelAsString": true
+          },
+          "description": "Access tier for specific share. Default is TransactionOptimized for GpV2 account and Premium for FileStorage account."
+        },
+        "accessTierChangeTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Indicates the last modification time for share access tier."
+        },
+        "accessTierStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Indicates if there is a pending transition for access tier."
+        },
+        "shareUsageBytes": {
+          "type": "integer",
+          "readOnly": true,
+          "description": "The approximate size of the data stored on the share. Note that this value may not include all recently created or recently resized files."
```

Comment:
@zfchen95 
As we get on the last meeting, we will have feature "Share status" GA soon. 
But I don't see it in this PR, When will add it?

Reply:
There is no share status. I believe you are referring to share tier

Reply:
OK, thanks!